submissions/Zakkarat/html-forms/index.html

```diff
@@ -0,0 +1,460 @@
+                    title="Enter your credit card number. Example: 1234 5678 9012 4567"
+                    minlength="16"
+                    maxlength="16"
+                    required
+                  /><span class="cc-number-img"></span>
+                </div>
+                <div class="field-container field-date">
+                  <h5 class="above-text">EXPIRY DATE</h5>
+                  <input
+                    type="text"
+                    inputmode="numeric"
+                    id="card"
+                    placeholder="MM/YY"
+                    class="text text-card"
+                    name="card"
+                    pattern="\d*"
+                    title="Example: 12/22"
+                    minlength="4"
+                    maxlength="4"
+                    required
+                  />
+                </div>
+                <div class="field-container field-code">
+                  <h5 class="above-text">CODE</h5>
+                  <input
+                    type="text"
+                    inputmode="numeric"
+                    id="card"
+                    class="text text-card"
+                    name="card"
+                    pattern="\d*"
+                    title="Example: 235"
+                    minlength="3"
+                    maxlength="3"
+                    required
+                  />
+                </div>
+              </div>
+            </div>
+          </section>
+          <section class="submit">
+            <div class="block-to-center submit-field">
+              <input type="checkbox" />
+              <div>
+                <h5 class="smallUploadText submit-text">
+                  I agree to the.
+                  <a href="#" class="underlined-a"> Terms & Conditions</a>
+                </h5>
+              </div>
+              <button class="upperMenuButton order">PLACE ORDER</button
+              ><button class="upperMenuButton">PRINT A QUOTE</button>
+            </div>
+          </section>
+        </form>
+      </div>
+    </main>
+    <footer class="footer">
+      <div class="footer-wrapper">
+        <div class="footer-text-container">
+          <h4 class="info-header">WHAT WE DO</h4>
+          <p class="range-para">
+            We provide a full range of language services including content
+            development, document, website and social media accounts
+            translations.
+          </p>
+        </div>
+        <div class="footer-text-container">
+          <h4 class="info-header">OUR MAIN OFFICE</h4>
+          <p class="range-para">Dubai, UAE</p>
+          <p class="range-para">P.O.Box: 502735</p>
+          <p class="range-para">
+            Phone: <a href="#" class="phone">+971 (0) 44278128</a>
+          </p>
+        </div>
+        <div class="footer-text-container">
+          <h4 class="info-header">CONNECT WITH US</h4>
+          <a href="#"><p class="range-para facebook">Facebook</p></a>
```

Comment:
Add some style on `:hover`

Reply:
Added only text color change due to my pseudo-element is a png.